polaris-migrator/src/migrations/replace-sass-lengths/tests/replace-sass-lengths.output.scss

```diff
@@ -0,0 +1,17 @@
+.my-class {
+  color: red;
+  padding-top: 13px;
+  padding: var(--p-space-0) var(--p-space-4) 2em 3px 4in;
+  padding-bottom: var(--p-space-0);
+  padding: var(--p-space-05) var(--p-space-05);
+  /* Expect to skip negative lengths */
```

Comment:
Should we be adding the same `POLARIS_MIGRATOR_COMMENT` comment here as well?

Reply:
@aaronccasanova and I adjusted the scope of this migration to skip negative values and sass functions. 

#### Reason for skipping negative values
We need to discuss as a team how to apply negative values with tokens. For example: 
```scss
padding: calc(-1 * var(--p-space-1));
padding: calc(-1 * var(--p-space-2));
padding: calc(-1 * var(--p-space-3));
```

> _**Note**: Using calc may be problematic as it's very easy to extend the calculation and thus it could make future migrations harder (ex. `calc(-1 * var(--p-space-3) + X / Y * Z)`)._

versus having dedicated negative tokens (which could be easier to migrate):

```scss
padding: var(--p-space--1);
padding: var(--p-space--2);
padding: var(--p-space--3);
```
#### Reason for skipping all sass functions
Handling edge cases for Sass functions (e.g. `rem()`, `calc()`, etc.) significantly increased the code complexity of this migration. We felt that it made more sense to tighten the scope knowing that we can create another migration to target those functions.